native/src/main/java/io/ballerina/stdlib/oracledb/parameterprocessor/OracleDBResultParameterProcessor.java

```diff
@@ -132,4 +136,34 @@ public BObject getBalStreamResultIterator() {
         }
         return struct;
     }
+
+    @Override
+    public Object convertXml(SQLXML value, int sqlType, Type type) throws ApplicationError, SQLException {
+        Utils.validatedInvalidFieldAssignment(sqlType, type, "SQL XML");
+        if (value != null) {
+            if (type.getTag() == TypeTags.XML_TAG) {
+                return XmlUtils.parse(value.getBinaryStream());
+            } else {
+                throw new ApplicationError("The ballerina type that can be used for SQL struct should be record type," +
```

Comment:
Is "should be **record type**" correct here?

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/10/commits/5c347d70db02692b3a10a12144d737eef71990de